Lib/test/test_syntax.py

```diff
@@ -626,6 +626,9 @@
     Traceback (most recent call last):
     SyntaxError: cannot assign to __debug__
 
+>>> (__debug__ := 'spam')
```

Comment:
Please, move this one after the check for `>>> __debug__ = 1`

Reply:
done, and I have added you as Co-authored-by